sources/views/tabletoolbar.js

```diff
@@ -0,0 +1,94 @@
+import {JetView} from "webix-jet";
+import {icons} from "../models/icons";
+
+export default class TableToolbar extends JetView {
+	constructor(app, name, data) {
+		super(app, name);
+		this._gridData = data;
+	}
+
+	config() {
+		const _ = this.app.getService("locale")._;
+		const datatable = {
+			view: "datatable",
+			localId: "settings:datatable",
+			scroll: "y",
+			select: true,
+			editable: true,
+			columns: [
+				{
+					id: "value",
+					header: _("Value"),
+					editor: "text",
+					fillspace: 2
+				},
+				{},
+				{
+					view: "richselect",
```

Comment:
`view` does not belong to the column configuration

Reply:
fixed